### PR TITLE
Add company:  Infuse AI

### PR DESCRIPTION
**Pull request by dhruvchandak30**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [ ] You are an employee of the company mentioned and confirm all included details are correct.
* [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

fixes: #1206

### CODE REVIEW

**Review by akhilbharti — COMMENTED** [review]
https://jobs.infuseai.io/
link does not exist 

image

**Comment by dhruvchandak30** [comment]
> https://jobs.infuseai.io/ link does not exist
> 
> image

Their Career's Page is not hosted also there is no other section for Careers. Should I leave that column blank or redirect to their Contact Page?

File: company-profiles/infuseai.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+We used Gitlab and Github for Development. Clubhouse and Status Hero for issue tracking. Airtable, Docsend, DocuSign, GSuite, Miro, Notion, Slack, Zendesk, Zoom for Communication and Documentation. Lastly, we used Attendancebot and Recruitee for Human Resource Management.
+
+## Office locations
+
+InfuseAI is headquartered in Taipei, Taiwan.
+
+## How to apply
+
+Please apply through [our jobs listing page](https://jobs.infuseai.io/).
```
Comment (KieranRobson) [inline company-profiles/infuseai.md:30]: Please find a place they host their jobs such as linkedin or change to a contact page.

**Review by KieranRobson — CHANGES_REQUESTED** [review]
Quick change to be made.

File: company-profiles/infuseai.md (hunk 1 above)
Comment (dhruvchandak30) [reply company-profiles/infuseai.md]: @KieranRobson  Done with the changes.

**Review by KieranRobson — APPROVED** [review]
Looks good to me! - Thanks

**Pull request closed without merging**